artemis-server/src/main/java/org/apache/activemq/artemis/core/server/management/impl/ManagementServiceImpl.java

```diff
@@ -719,10 +671,18 @@ public void activated() {
             }
          }
       });
+
+      started = true;
    }
 
    @Override
    public synchronized void stop() throws Exception {
+      if (!started) {
```

Comment:
Similar to above this doesn't really prevent concurrent calls to stop for overlapping so this also needs to be idempotent.  There is also the concern that start / stop calls can run concurrently now which could leave this in an odd state if not handled.